code/modules/clothing/under/jobs/medsci.dm

```diff
@@ -116,3 +116,18 @@
 	name = "moebius psychologist's turtleneck"
 	icon_state = "psychturtle"
 	item_state = "b_suit"
+
+/obj/item/clothing/under/rank/paramedic
+	desc = "Jumpsuit of Moebius Medical paramedics. It's made with reinforced fiber to offer more protection for recovery operators."
+	name = "moebius paramedic's jumpsuit"
+	icon_state = "paramedic"
+	item_state = "paramedic"
+	permeability_coefficient = 0.50
+	armor = list(
```

Comment:
Armor is not added to uniform.

Reply:
You know that every uniform for every department has armor values?